tailwind.config.js

```diff
@@ -13,13 +13,26 @@ module.exports = {
       fontFamily:{
          sans:["Neufile Grotesk"] // called it sans for it to act as default
       },
+      letterSpacing:{
+        tight:"-1%" // this a letter spacing -1 percentage
+      },
+      lineHeight:{
```

Comment:
remove the LineHeight configurations, because it is done in card #012. thus it will solve having conflicts in the config file.